internal/helm/repository/oci_chart_repository.go

```diff
@@ -153,7 +164,7 @@ func (r *OCIChartRepository) getTags(ref string) ([]string, error) {
 	// Retrieve list of repository tags
 	tags, err := r.RegistryClient.Tags(strings.TrimPrefix(ref, fmt.Sprintf("%s://", registry.OCIScheme)))
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("could not get tags for %q: %s", ref, err)
```

Comment:
```suggestion
		return nil, fmt.Errorf("could not fetch tags for %q: %s", ref, err)
```
This differentiates it from the error above to simplify associating the error with the line of code.